src/PointerEvent.js

```diff
@@ -87,6 +87,35 @@ function PointerEvent(inType, inDict) {
   e.pointerType = inDict.pointerType || '';
   e.hwTimestamp = inDict.hwTimestamp || 0;
   e.isPrimary = inDict.isPrimary || false;
+
+  var rect;
+  function boundingClientRect() {
+    if (rect) {
+      return rect;
+    }
+    return (rect = inDict.target.getBoundingClientRect());
+  }
+
+  // CSSOM View Module's extension to mouse events
+  if (inDict.offsetX === undefined) {
+    Object.defineProperty(e, 'offsetX', {
+      get: function() {
+        return e.pageX - boundingClientRect().left;
```

Comment:
On an element with 10px padding, what value should `offsetX` have when the pointer is above to the top left corner of the element? If my test is correct, currently it has 0, which seems correct to me. Same for the top right corner: On an element with 10px padding on all sides, 130px width, the top left corner has offsetX 149.